Templates/JenkinsPipeline/README.md

```diff
@@ -0,0 +1,132 @@
+# Jenkins Pipeline template
+
+This template provides an Jenkinsfile [Jenkinsfile](Jenkinsfile) definition to setup a Jenkins Multibranch pipeline for applications managed in any Git provider.
```

Comment:
'for applications managed in any Git provider.' -> 'compatible with any Git provider'.